tests/test_component.py

```diff
@@ -55,6 +55,16 @@ def test_requirement_from_version_constraints(self):
                     "sha": "0000000000000000000000000000000000000007",
                     "series": "first",
                 },
+                {
+                    "version": "2.0.0-rc.1",
+                    "sha": "0000000000000000000000000000000000000007",
+                    "series": "first",
+                },
+                {
+                    "version": "2.0.0-rc.2",
+                    "sha": "0000000000000000000000000000000000000007",
+                    "series": "first",
+                },
                 {
                     "version": "2.0.0",
                     "sha": "0000000000000000000000000000000000000008",
```

Comment:
Maybe update `constraints_test_cases` to ensure works as a constraint.

Reply:
Done.